dds/DCPS/RTPS/RtpsCore.idl

```diff
@@ -287,6 +287,14 @@ module OpenDDS {
     const ParameterId_t PIDMASK_VENDOR_SPECIFIC = 0x8000;
     const ParameterId_t PIDMASK_INCOMPATIBLE = 0x4000;
 
+    typedef unsigned long ParticipantFlagsBits_t;
```

Comment:
```suggestion
    typedef unsigned long OpenDDSParticipantFlags_t;
```
Possible future naming collision if spec changes... we could use modules but it doesn't seem necessary here.